src/Lecture2.hs

```diff
@@ -212,7 +289,12 @@ verify that.
 [1,2,3,4,7]
 -}
 merge :: [Int] -> [Int] -> [Int]
-merge = error "TODO"
+merge firstList secondList = case (firstList, secondList) of
+    ([], y) -> y
+    (x, []) -> x
+    (x : xs, y : ys) -> if x < y
+                        then x : merge xs (y : ys)
+                        else y : merge (x : xs) ys
```

Comment:
Absolutely correct solution! 💎 
You can make it more performant by checking if both heads of the given lists are equal. In that case, you're able to reduce the size of each list by 1 on each recursive call instead of reducing the size of only a single list ✂️

Reply:
I know the course was a year ago, yet you're still here looking at a random person's code on the Internet for free.
Thank you doing this!

Reply:
You're welcome 🤗 

Happy to help people who want to learn new things and become better engineers 🙂